test/gtest/uct/test_atomic_key_reg_rdma_mem_type.cc

```diff
@@ -7,24 +7,30 @@
 #include "test_amo.h"
 
 class uct_atomic_key_reg_rdma_mem_type : public uct_amo_test {
+protected:
+    bool check_rdma_memory()
+    {
+        FOR_EACH_ENTITY(iter) {
+            if (!((*iter)->md_attr().alloc_mem_types &
+                  UCS_BIT(UCS_MEMORY_TYPE_RDMA))) {
+                return false;
+            }
+        }
+        return true;
+    }
 };
 
 UCS_TEST_SKIP_COND_P(uct_atomic_key_reg_rdma_mem_type, fadd64,
-                     !check_atomics(UCS_BIT(UCT_ATOMIC_OP_ADD), FOP64))
+                     !check_atomics(UCS_BIT(UCT_ATOMIC_OP_ADD), FOP64) ||
+                             !check_rdma_memory())
```

Comment:
weird alignment

Reply:
Agree, I can add () in order for clang-format to align it, otherwise this is what clang-format is enforcing.
We can also just align it manually

Reply:
can ignore clang suggestion here